src/util/events.js

```diff
@@ -8,7 +8,14 @@ export function getFilteredEvents(events, filters, zipcodes) {
     return events;
   }
 
-  const zipCodesLength = (Object.keys(zipcodes) || []).length;
+  const zipcodesLength = (Object.keys(zipcodes) || []).length;
+  const validZipcode = zipcodesLength && !!zipcodes[filters.zipcode];
+
+  // We do not yet have zipcodes to filter against, or the selected
+  // zipcode is not valid...so there can be no valid events. Bail early.
+  if (filters.zipcode && (!zipcodes || !zipcodesLength || !validZipcode)) {
```

Comment:
When would this happen? If the zipcode JSON file is empty/missing? IMO we should throw a useful error here rather than fail silently.